.github/workflows/pull-request.yml

```diff
@@ -25,6 +25,8 @@ jobs:
           distribution: 'temurin' 
           java-version: '11'
           cache: 'maven'
+      - name: Local Tests
```

Comment:
The goal is to verify ricochet compliance ? We might want to remove the test on osc API, no ?

Reply:
the goal, is to enable tests to run without the need for an auth at Outscale.
We could remove the `Example tests`, as both tests are similar. But as ricochet is still in its early stage, I guess it's better to keep both tests for a few time, at last until we decide when to test on our API, and when to use ricochet tests.